### PR TITLE
Revise Options

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/Options/en.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If you are a guest, these options will be shown:
 | `Remember Username` | If checked, the game client will remember the entered username. | Checkbox | `Enabled` |
 | `Remember Password` | If checked, the game client will remember the entered password. This will also keep you signed in to the game client. | Checkbox | `Disabled` |
 | `Sign In` | Sign in to osu! with the given credentials. Pressing `Enter` in the username or password forms also works. | Button |  |
-| `Create an account` | Redirects you to [Create Account](https://osu.ppy.sh/p/register) using your default web browser. | Button |  |
+| `Create an account` | Opens the account registration screen. | Button |  |
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
| `Create an account` | Opens the [account registration](/wiki/Registration) screen. | Button |  |
```

File: wiki/Options/en.md [hunk 2]
```diff
@@ -278,12 +278,12 @@ This section is about input peripherals.
 | Name | Description | Type | Default |
 | --- | --- | --- | --- |
 | `Sensitivity` | Adjust the sensitivity of the mouse cursor. If changed, it may automatically enable `Raw Input`. | Slider | `1x` |
-| `Raw Input` | Read mouse/tablet positional values directly from the hardware, without any post-processing (this is to ignore mouse acceleration). | Button | Disabled |
-| `Map absolute raw input to the osu! window` | Confine input devices with absolute positioning (e.g. pen tablets) to the game client window only. | Button | Enabled |
+| `Raw input` | Read mouse/tablet positional values directly from the hardware, without any post-processing (this is to ignore mouse acceleration). | Button | `Disabled` |
+| `Map absolute raw input to the osu! window` | Confine input devices with absolute positioning (e.g. pen tablets) to the game client window only. | Button | `Enabled` |
```
Comment (MegaApplePi) [inline wiki/Options/en.md]:
```suggestion
| `Map absolute raw input to the osu! window` | Confine input devices with absolute positioning (e.g. pen tablets) to the game client window only. | Button | `Disabled` |
```

**Comment by cl8n** [comment]
might want to compare to user config file and see if any option names there need updating too

**Comment by MegaApplePi** [comment]
Other than letter casing and minor variance in some of the option names, the list in the User Configuration File looks to be the same.

File: wiki/Options/en.md [hunk 3]
```diff
@@ -350,17 +351,20 @@ This section is about chat, spectators, multi, and osu!direct.
...
+| `Show notification popups when friends change status` | Display a notification, whenever your friend goes online or offline. | Checkbox | `Enabled` |
 | `Allow multiplayer game invites from all users` | Allow multi game invites from anyone. Disabling this will limit multiplayer invites to friends only. | Checkbox | `Enabled` |
 
 ### Integration
 
 | Name | Description | Type | Default |
 | --- | --- | --- | --- |
+| `Discord Rich Presence` | Provides rich presence data to Discord. | Checkbox | `Enabled` |
 | `Integrate with Yahoo! status display` | Your Yahoo! messenger will show the song you are currently playing or listening to. This needs to be set from the website. | Checkbox | `Disabled` |
 | `Integrate with MSN Live status display` | Your Windows Live Messenger will show the song you are currently playing or listening to. This needs to be set from the osu!website. | Checkbox | `Disabled` |
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Options/en.md]: what does it mean anyway? that's not an option on the website

[the remark this answers is not in the document]
Comment (MegaApplePi) [reply wiki/Options/en.md]: Huh, iirc you needed to set the Skype field for this to work. Then again, you also needed MSN Live, which appearently will update to Skype or something.

**Pull request merged**